docs/toml.md

```diff
@@ -1708,10 +1712,16 @@ Træfik can be configured to use Amazon ECS as a backend configuration:
 
 # ECS Cluster Name
 #
+# Deprecated - Please use Clusters
+#
+# Cluster = "default"
+
+# ECS Clusters Name
+#
 # Optional
-# Default: "default"
+# Default: ["default"]
 #
-Cluster = "default"
+Clusters = ["default"]
```

Comment:
In fact, the current branch seems to be out-of-date with the master.
Only he file `docs/toml.md` is up-to-date that's why these differences appear.